test/src/unit-capi-dense_array.cc

```diff
@@ -869,6 +869,7 @@ void DenseArrayFx::write_dense_subarray_2D_with_cancel(
     REQUIRE(rc == TILEDB_OK);
     // Cancel it immediately.
     if (i < num_writes - 1) {
+      std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
This was added because it seems we have an issue in cancellable tasks when the write throws before it is submitted...